addons/web/static/src/views/calendar/calendar_model.js

```diff
@@ -60,6 +63,9 @@ export class CalendarModel extends Model {
                 ),
             },
         };
+        if (params.state?.multiCreate?.timeRange) {
+            this.data.multiCreateTimeRange = params.state?.multiCreate.timeRange;
+        }
```

Comment:
```suggestion
        if (params.state?.multiCreate) {
            this.data.multiCreate = params.state.multiCreate;
        }
```

maybe consider renaming into multiCreateState ? Not sure